frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -67,38 +68,46 @@ const Sidebar = (props: LinkProps) => {
           <MenuItem onClick={handleClose}>
             <Link to='/assistants'>Assistants</Link>
           </MenuItem>
+          <MenuItem onClick={handleClose}>
+            <Link to='/settings'>Settings</Link>
+          </MenuItem>
+          <MenuItem onClick={handleClose}>
+            <Link to='/signout'>Signout</Link>
+          </MenuItem>
         </Menu>
       </div>
-      <NavWrapper>
-        <StyledUL>
-          <li>
-            <StyledLink to='/properties'>
-              <h4>Properties</h4>
-            </StyledLink>
-          </li>
-          <li>
-            <StyledLink to='/guests'>
-              <h4>Guests</h4>
-            </StyledLink>
-          </li>
-          <li>
-            <StyledLink to='/assistants'>
-              <h4>Assistants</h4>
-            </StyledLink>
-          </li>
-        </StyledUL>
-      </NavWrapper>
-      <SettingsWrapper>
-        <StyledLink to='/settings'>
-          <h4>Settings</h4>
-        </StyledLink>
+      <NavBar>
+        <NavWrapper>
+          <StyledUL>
+            <li>
+              <StyledLink to='/properties'>
+                <h4>Properties</h4>
+              </StyledLink>
+            </li>
+            <li>
+              <StyledLink to='/guests'>
+                <h4>Guests</h4>
+              </StyledLink>
+            </li>
+            <li>
+              <StyledLink to='/assistants'>
+                <h4>Assistants</h4>
+              </StyledLink>
+            </li>
+          </StyledUL>
+        </NavWrapper>
+        <SettingsWrapper>
+          <StyledLink to='/settings'>
+            <h4>Settings</h4>
+          </StyledLink>
 
-        <Button
-          onClick={() => logOut()}
-          className='button-sign-out'
-          text='Sign Out'
-        />
-      </SettingsWrapper>
+          <SpecialButton
```

Comment:
I thought that it would be easier from a stylistic/functional perspective to use a separate button, and it wasn't until I finished revamping SpecialButton that I noticed I could have used the regular Button for it.